rmgpy/reaction.pxd

```diff
@@ -113,7 +114,9 @@ cdef class Reaction:
 
     cpdef bint can_tst(self) except -2
 
-    cpdef calculate_microcanonical_rate_coefficient(self, np.ndarray e_list, np.ndarray j_list, np.ndarray reac_dens_states, np.ndarray prod_dens_states=?, double T=?)
+    cpdef calculate_microcanonical_rate_coefficient(self, np.ndarray e_list, np.ndarray j_list,
+                                                    np.ndarray reac_dens_states, np.ndarray prod_dens_states=?,
+                                                    double T=?)
```

Comment:
This is just whitespace changes, right? If so can you move it to a separate minor commit?